docs/content/s3.md

```diff
@@ -3359,10 +3359,13 @@ endpoint = s3.nl-ams.scw.cloud
 access_key_id = SCWXXXXXXXXXXXXXX
 secret_access_key = 1111111-2222-3333-44444-55555555555555
 region = nl-ams
-location_constraint =
+location_constraint = nl-ams
 acl = private
-server_side_encryption =
-storage_class =
+upload_cutoff = 5M
+chunk_size = 5M
+copy_cutoff = 5M
+list_url_encode = true
```

Comment:
You don't need to put list_url_encode or use_already_exists here as they are set in a quirk now - I'd remove them from here

Reply:
true. done :heavy_check_mark: